core/linux-armv7-rc/PKGBUILD

```diff
@@ -17,10 +17,14 @@ pkgrel=2
 arch=('armv7h')
 url="http://www.kernel.org/"
 license=('GPL2')
-makedepends=('xmlto' 'docbook-xsl' 'kmod' 'inetutils' 'bc' 'git' 'uboot-tools' 'vboot-utils' 'dtc')
+makedepends=('xmlto' 'docbook-xsl' 'kmod' 'inetutils' 'bc' 'git' 'uboot-tools' 'vboot-utils' 'dtc' 'wireless-regdb')
 options=('!strip')
 source=("https://git.kernel.org/torvalds/t/${_srcname}.tar.gz"
         "http://rcn-ee.com/deb/sid-armhf/v${_rcver}.0-rc${_rcnrc}-${_rcnrel}/patch-${_rcver}-rc${_rcnrc}-${_rcnrel}.diff.gz"
```

Comment:
Firmware is being provided in this patch.

Reply:
Which patch are you refering to? Currently, it is broken.

Reply:
Don't build inside the git clone, the 'git apply' statements for the patches won't work if you do that.

Reply:
@kmihelich if you refer to the rcn-ee.com , please have a look, this patch does not provide any firmware. You can check i.e.: https://rcn-ee.com/deb/sid-armhf/v5.16.11-armv7-x15/patch-5.16.11-armv7-x15.diff.gz